crawl-ref/source/dat/database/monspeak.txt

```diff
@@ -5042,6 +5083,28 @@ _Maggie_rare_
 @The_monster@ shouts @at_foe@, "Leave me alone!"
 
 @The_monster@ mutters to herself, "Master Lance won't take me back. This is the way forward."
+%%%%
+Maggie killed
+
+w:10
```

Comment:
don't need to write w:10, that's already the default weight of a speech line

Reply:
Thanks, I wasn’t aware